spec.bs

```diff
@@ -2823,6 +2824,12 @@ might choose to require a [=k-anonymity=] threshold of fifty users over a seven
 will maintain the count over the chosen duration and compare the count to the chosen [=k-anonymity=]
 threshold when responding to [=query k-anonymity count=].
 
+The [=user agent=] should maintain a <dfn>k-anonymity cache</dfn> as a [=map=]

```

Comment:
Should is interpreted as optional. Is this cache indeed optional / an optimization?

Reply:
Changed to must